source/direct_debit/index.html.md.erb

```diff
@@ -255,96 +133,197 @@ Required arguments:
  "amount"
  "description"
```

Comment:
description is now optional

Reply:
maybe it was good if the field specification was in the same format as we've given for the create mandate request.

Reply:
Thanks for the 'description' info!
I'd definitely like to make the field specifications consistent, but it may be too much for this iteration. I'll check at the 2i stage - but we'll make everything consistent in the next iteration in any case.